bcgov/moh-prime
# PRIME-1562 .Net Core 5 Upgrade

Merging of this PR will not occur until OCP4 has been deployed, but the code changes are up for review even though the PR won't build in OCP3 due to an issue with the migrations.  This can be tested locally.

When it eventually becomes time to upgrade local development:

1. Install the [.Net Core 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) installer and install it on your local machine
2. Run `dotnet --version` to verify version
3. Run `dotnet tool update --global dotnet-ef`
4. Run `dotnet ef --version` to verify version
5. Delete `/bin` and `/obj` folders
6. Run `dotnet build` which will run a restore as well as the build
7. Rebuild the containers
